cmd/testscript/main.go

```diff
@@ -273,11 +273,21 @@ func run(runDir, filename string, update bool, verbose bool, envVars []string) e
 	if len(envVars) > 0 {
 		addSetup(func(env *testscript.Env) error {
 			for _, v := range envVars {
-				if v == "WORK" {
+				varName := v
+				if i := strings.Index(v, "="); i >= 0 {
+					varName = v[:i]
+				} else {
+					v = fmt.Sprintf("%s=%s", v, os.Getenv(v))
+				}
+				switch varName {
+				case "":
+					// Nonense variable
```

Comment:
Maybe this would be better as an error, as it's almost certainly a mistake.

Reply:
Yep, fair point.